cuetojson/values.go

```diff
@@ -0,0 +1,53 @@
+package cuetojson
+
+import (
+	"cuelang.org/go/cue"
+	"dagviz/dag"
+)
+
+func AppendValuesToDag(root *dag.Root, programs []CueProgram) {
+	for _, program := range programs {
+		addNode(root, nil, program.Values, nil)
+	}
+	root.PrintDag(" --> ")
+}
+
+func addNode(root *dag.Root, node *dag.Node, value *cue.Value, prev *cue.Iterator) {
+	iterator, err := value.Fields()
```

Comment:
Why you didn't used function [`Walk`](https://cuetorials.com/go-api/basics/traversing/#walking-a-value) to explore value?

Reply:
The use of Walk will be a subject of the next sprint (thanks to the call we had this morning)